openage/__main__.py

```diff
@@ -56,6 +56,10 @@ def main(argv=None):
                             help=("upon throwing an exception a debug break is "
                                   "triggered. this will crash openage if no "
                                   "debugger is present"))
+    global_cli.add_argument(
+        "--add-dll-search-path", action='append', dest='dll_paths',
+        help="(Windows only) provide additional DLL search path")
+
```

Comment:
If the argument is only needed on Windows, we could add the `if sys.platform == 'win32'` platform check here too.

Reply:
Added. Thanks!